packages/bootstrap-chat/src/FormResponse.jsx

```diff
@@ -98,6 +98,14 @@ class FormResponse extends React.Component {
                     return { responses: new_res };
                   });
                 }}
+                onKeyPress={(e) => {
+                  console.log(e);
+                  if (e.key === "Enter") {
+                    tryMessageSend();
+                    e.stopPropagation();
+                    e.nativeEvent.stopImmediatePropagation();
+                  }
+                }}
```

Comment:
So I believe this onKeyPress handler should be added instead to L118 (L126 after this addition).

```jsx
<FormControl
    type="text"
    style={{ fontSize: "16px" }}
```